### ISSUE TITLE
lore: tutorial.xhtml can't be viewed in Mozilla 1.5

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7600]
|<img alt="maksim's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| maksim reported|
|-|-|
|Trac ID|trac#376|
|Type|defect|
|Created|2003-10-30 14:11:49Z|

<details><summary>Searchable metadata</summary>

```
trac-id__376 376
type__defect defect
reporter__maksim maksim
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1067523109000000 1067523109000000
changetime__1469093560786071 1469093560786071
version__ 
owner__ 
cc__maksim
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="maksim's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|maksim commented|
|-|-|

```
#!html
<pre>
When trying to open howto/tutorial.xhtml in a latest Mozilla it gives the
following error:
XML Parsing Error: duplicate attribute Location:
file:///D:/Python23/lib/site-packages/TwistedDocs/howto/tutorial.xhtml Line
Number 506, Column 60:&lt;a class="py-listing" href="listings/finger/finger/tap.py"
class="py-listing">finger/tap.py&lt;/a>
-----------------------------------------------------------^
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="moshez's avatar" src="https://avatars.githubusercontent.com/u/422703?s=50" width="50" height="50">](https://github.com/moshez)|@moshez commented|
|-|-|

```
#!html
<pre>
/cvs/Twisted/doc/howto/tutorial.xhtml,v  &lt;--  tutorial.xhtml
new revision: 1.6; previous revision: 1.5
Fixed
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub commented|
|-|-|

In changeset e94222cb3a388b4dc45e15ba44c42cf0d0e84106
```
#!CommitTicketReference repository="" revision="e94222cb3a388b4dc45e15ba44c42cf0d0e84106"
Merge pull request #376 from twisted/8638-einarfd-ssh-transport

Author: einarfd
Reviewer: rodrigc
Fixes: #8638

Port twisted.conch.ssh.transport to Python 3
```
